demisto_sdk/commands/common/tools.py

```diff
@@ -3530,47 +3530,52 @@ def get_display_name(file_path, file_data={}) -> str:
     :rtype: ``str``
     :return The display name
     """
-    if not file_data:
-        file_extension = os.path.splitext(file_path)[1]
-        if file_extension in [".yml", ".json"]:
-            file_data = get_file(file_path)
-
-    if "display" in file_data:
-        name = file_data.get("display", None)
-    elif "layout" in file_data and isinstance(file_data["layout"], dict):
-        name = file_data["layout"].get("id")
-    elif "name" in file_data:
-        name = file_data.get("name", None)
-    elif "TypeName" in file_data:
-        name = file_data.get("TypeName", None)
-    elif "brandName" in file_data:
-        name = file_data.get("brandName", None)
-    elif "id" in file_data:
-        name = file_data.get("id", None)
-    elif "trigger_name" in file_data:
-        name = file_data.get("trigger_name")
-    elif "rule_name" in file_data:
-        name = file_data.get("rule_name")
+    _file_path = Path(file_path)
+    _file_data: dict = file_data or {}
 
+    if not file_data and _file_path.suffix in (".yml", ".yaml", ".json"):
+        _file_data = get_file(_file_path)
+
+    if "display" in _file_data:
+        return _file_data["display"]
     elif (
-        "dashboards_data" in file_data
-        and file_data.get("dashboards_data")
-        and isinstance(file_data["dashboards_data"], list)
+        "layout" in _file_data
+        and isinstance(_file_data["layout"], dict)
+        and "id" in _file_data["layout"]
     ):
-        dashboard_data = file_data.get("dashboards_data", [{}])[0]
-        name = dashboard_data.get("name")
-
+        return _file_data["layout"]["id"]
+    elif "name" in _file_data:
+        return _file_data["name"]
+    elif "TypeName" in _file_data:
+        return _file_data["TypeName"]
+    elif "brandName" in _file_data:
+        return _file_data["brandName"]
+    elif "reputationCommand" in _file_data and "details" in _file_data:
+        return _file_data["details"]
```

Comment:
GitHub kind of messed up the diffs, so these lines (which weren't there before) are the fix.
(The display name of reputation content items is within the `details` field. Tested it by creating a new item on the UI.)